tools/gn

```diff
@@ -324,8 +328,12 @@ def to_gn_args(args):
   # No cross-compilation on Windows (for now). Use host toolchain that
   # matches the bit-width of the target architecture.
   if sys.platform.startswith(('cygwin', 'win')) and args.target_os != 'win':
-    gn_args['host_cpu'] = cpu_for_target_arch(gn_args['target_cpu'])
-    gn_args['target_cpu'] = cpu_for_target_arch(gn_args['target_cpu'])
+    cpu = cpu_for_target_arch(gn_args['target_cpu'])
+    # We explicitly allow arm64 native build.
+    if gn_args['host_cpu'] == 'arm64' and gn_args['target_cpu'] == 'arm64':
+      cpu = 'arm64'
```

Comment:
Looks like the `gn_args['host_cpu']` check is failing since it isn't for Android builds on Windows:
https://ci.chromium.org/ui/p/flutter/builders/try/Windows%20Android%20AOT%20Engine/43322/overview

Reply:
Good catch. I wrongly assumed it *always* existed. I changed that to `gn_args.get('host_cpu')` and added a comment for that.